### PR TITLE
Fixes #15648 - do not pause on CandlepinEvent errors

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test]

**Comment by thomasmckay** [comment]
apj

**Comment by jlsherrill** [comment]
[test]

File: app/lib/actions/candlepin/listen_on_candlepin_events.rb [hunk 1]
```diff
@@ -180,10 +180,11 @@ def finish_service
...
         begin
           output[:connection] = "Connected"
-          on_event(event)
+          Actions::Candlepin::ImportPoolHandler.new(Rails.logger).handle(event)
+          output[:last_message] = "#{event.message_id} - #{event.subject}"
+          output[:messages] = event.message_id
         rescue => e
-          error!(e)
-          raise e
+          output[:last_event_error] = e.message
```
Comment (ehelms) [inline app/lib/actions/candlepin/listen_on_candlepin_events.rb]: I suppose this implies if there are multiple errors coming through, you will only know the last?

Comment (jlsherrill) [inline app/lib/actions/candlepin/listen_on_candlepin_events.rb]: indeed, i guess i could add some more logging

Comment (jlsherrill) [inline app/lib/actions/candlepin/listen_on_candlepin_events.rb]: added

**Comment by jlsherrill** [comment]
are labels broken?

**Comment by ehelms** [comment]
ACK

**Pull request merged**